src/main/scala/djinni/Main.scala

```diff
@@ -216,27 +216,33 @@ object Main {
       opt[Boolean]("skip-generation").valueName("<true/false>").foreach(x => skipGeneration = x)
         .text("Way of specifying if file generation should be skipped (default: false)")
 
-      note("\nIdentifier styles (ex: \"FooBar\", \"fooBar\", \"foo_bar\", \"FOO_BAR\", \"m_fooBar\")\n")
-      identStyle("ident-java-enum",      c => { javaIdentStyle = javaIdentStyle.copy(enum = c) })
-      identStyle("ident-java-field",     c => { javaIdentStyle = javaIdentStyle.copy(field = c) })
-      identStyle("ident-java-type",      c => { javaIdentStyle = javaIdentStyle.copy(ty = c) })
-      identStyle("ident-cpp-enum",       c => { cppIdentStyle = cppIdentStyle.copy(enum = c) })
-      identStyle("ident-cpp-field",      c => { cppIdentStyle = cppIdentStyle.copy(field = c) })
-      identStyle("ident-cpp-method",     c => { cppIdentStyle = cppIdentStyle.copy(method = c) })
-      identStyle("ident-cpp-type",       c => { cppIdentStyle = cppIdentStyle.copy(ty = c) })
-      identStyle("ident-cpp-enum-type",  c => { cppTypeEnumIdentStyle = c })
-      identStyle("ident-cpp-type-param", c => { cppIdentStyle = cppIdentStyle.copy(typeParam = c) })
-      identStyle("ident-cpp-local",      c => { cppIdentStyle = cppIdentStyle.copy(local = c) })
-      identStyle("ident-cpp-file",       c => { cppFileIdentStyle = c })
-      identStyle("ident-jni-class",      c => { jniClassIdentStyleOptional = Some(c)})
-      identStyle("ident-jni-file",       c => { jniFileIdentStyleOptional = Some(c)})
-      identStyle("ident-objc-enum",       c => { objcIdentStyle = objcIdentStyle.copy(enum = c) })
-      identStyle("ident-objc-field",      c => { objcIdentStyle = objcIdentStyle.copy(field = c) })
-      identStyle("ident-objc-method",     c => { objcIdentStyle = objcIdentStyle.copy(method = c) })
-      identStyle("ident-objc-type",       c => { objcIdentStyle = objcIdentStyle.copy(ty = c) })
-      identStyle("ident-objc-type-param", c => { objcIdentStyle = objcIdentStyle.copy(typeParam = c) })
-      identStyle("ident-objc-local",      c => { objcIdentStyle = objcIdentStyle.copy(local = c) })
-      identStyle("ident-objc-file",       c => { objcFileIdentStyleOptional = Some(c) })
+      note("\n\nIdentifier styles (ex: \"FooBar\", \"fooBar\", \"foo_bar\", \"FOO_BAR\", \"m_fooBar\")")
+
+      note("\nC++ options:")
+      identStyle("ident-cpp-enum",        "FOO_BAR" ,    c => { cppIdentStyle = cppIdentStyle.copy(enum = c) })
```

Comment:
My understanding is that the default value parameter in here only affects the documentation, but not the actual default value. So if I wanted to change the actual default value of a parameter, it would not be enough to adjust this parameter here. Is that correct?

Reply:
good point!
 created #33 to cover that